apps/govquests-frontend/next.config.mjs

```diff
@@ -1,4 +1,9 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
-
+// TODO: DOMAIN JUST FOR TESTING, SHOUDL BE REMOVED OP-298
```

Comment:
We can also put NextImage in a Imagine component we control, and use the regular ing tag within there until we know which domains we'd add here